clickhouse/db-system-dictionaries.yml

```diff
@@ -0,0 +1,42 @@
+type: db
+data:
+  query: SELECT * FROM dictionaries;
+  dbUrl: jdbc:clickhouse://${SPM_MONITOR_CLICKHOUSE_DB_HOST_PORT}/system
+  dbDriverClass: ru.yandex.clickhouse.ClickHouseDriver
+  dbUser: ${SPM_MONITOR_CLICKHOUSE_DB_USER}
+  dbPassword: ${SPM_MONITOR_CLICKHOUSE_DB_PASSWORD}
+
+observation:
+  - name: dictionaries
+    metricNamespace: clickhouse
+
+    metric:
+      - name: dict.name
+        source: name
+        type: text
+        send: false
+      - name: dict.bytes.allocated
+        source: bytes_allocated
+        type: long_gauge
+        label: Bytes allocated
+        description: The amount of RAM used by the dictionary.
+      - name: dict.hit.rate
```

Comment:
Rate itself is useless metric for us, we need to calculate it on our backend at query time. No way to collect lookups and hits as separate metrics?